releases/release-1.18/README.md

```diff
@@ -1,3 +1,9 @@
+---
+title: "Kubernetes Release 1.18"
+weight: 98
```

Comment:
I'm not intimately familiar with the weights.
Was there a reason for choosing 98?

Reply:
Yup, We have several other side menu tabs that need to be shown below the Release Tab. Hence going with weight 98.

Reply:
Thanks!